examples/sky-get-info-ex.py

```diff
@@ -0,0 +1,10 @@
+from pyfreshintellivent.pyfreshintellivent.FreshIntellivent import Sky
+p = Sky("Fan adress","authcode from sky-auth-ex",False)
```

Comment:
Don't need to include False + spelling.
```suggestion
p = Sky("Fan address", "authcode from sky-auth-ex")
```